lesson01/assignment/scripts.js

```diff
@@ -29,5 +29,120 @@ document. addEventListener("DOMContentLoaded", function(){
     // Clear the form field so the user can try again
     inputField.value = ''
   })
+})
+
+document. addEventListener("DOMContentLoaded", function(){
+
+  // Find the example form, and handle its submit event
+  const form1 = document.querySelector('#form1')
+
+  form1.addEventListener('submit', function(event){
+    // Prevent the form from trying to submit to a server
+    event.preventDefault()
+
+    // An array that we check based on user input
+    const people = [
+    {name: 'Victor', age: 30},
+    {name: 'Gail', age: 28},
+    {name: 'John', age: 31},
+    {name: 'Nhi', age: 60},
+    {name: 'Daniel', age: 25},
+    {name: 'Du', age: 70}
+    ];
+
+    const inputField = document.querySelector('#form1 .input'); //take the users information entered
+    const userInput = inputField.value; //converting user input into a constant called userInput
+    const person = people.find(({name}) => name === userInput); //look in the array and see if there is a name object that corresponds with the users input
+ 
+    // output the array item quantity based on user input
+    document.querySelector('#form1 .output').innerHTML = person.age; //return the age of the person found
```

Comment:
This will a throw an error if person is not found. Consider validating the existence of `person` before moving on, like:
```
if(person){
 document.querySelector('#form1 .output').innerHTML = person.age;
document.querySelector('#form1 .name').innerHTML = userInput;
}else{
 document.querySelector('#form1 .output').innerHTML = "Sorry we couldn't find that person"
}
```